docs/dev/cli/sauce-connect-proxy.md

```diff
@@ -28,19 +28,53 @@ __Shorthand__: `-k`<br/>
 __Description__: Sets the local path to a YAML file containing a Sauce Connect Proxy configuration. An example YAML configuration file, `config.yaml`, is included for your reference as part of the Sauce Connect Proxy download package. We recommend using a YAML configuration file in production environments, rather than command-line options, as it facilitates tracking configuration changes, managing tunnel-domains and direct-domains options (which can get very long), and securing Sauce Connect Proxy credentials with tighter access control over the config file.<br/>
 __Shorthand__: `-c`
 
+
 ---
 ### `--no-remove-colliding-tunnels`
-__Description__: Make this tunnel a part of the High Availability Sauce Connect Proxy Tunnel Pool. For more info, see [High Availability Setup](/secure-connections/sauce-connect/setup-configuration/high-availability). <br/>
+__Description__: The tunnel is a part of the High Availability Sauce Connect Proxy Tunnel Pool. For more info, see [High Availability Setup](/secure-connections/sauce-connect/setup-configuration/high-availability). <br/>
+
+:::warning
+This option is aliased to `--tunnel-pool` in versions starting from v4.7.0. It will be removed in the future!
```

Comment:
wording. not sure how to change this, but this message should share a common format with other deprecated flags.

eg. Python docs: https://docs.python.org/3/library/http.client.html#http.client.HTTPResponse.geturl
Deprecated since version 4.7.0: `--no-remove-colliding-tunnels` has been deprecated in favor of `--tunnel-pool`.